kubernetes/skyring/values.yaml

```diff
@@ -0,0 +1,35 @@
+# Default values for skyring.
+# This is a YAML-formatted file.
+
+SeedReplicaCount: 2
+WorkerReplicaCount: 4
+
+image:
+  repository: ianskebba/skyring-node
```

Comment:
can we use one of the images from [quay](https://quay.io/repository/esatterwhite/skyring)? `7-latest` probably the most recent

Reply:
Is there a version on quay that has the mongo dependency?